### DISCUSSION TITLE
Approach for the title page

### DISCUSSION THREAD

**Discussion by m-kru** [typst/typst#4089] [Q&A]
I am wondering what is the typst approach for the title page. It looks like there is no separate topic for title pages in the typst tutorial. Some typesetting and markdown systems generate the title page automatically based on the provided meta data (title, author, date, etc.). This is nice if the default title page layout is what you want. However, creating custom title pages is a nightmare (my experience with latex, groff, and asciidoc). What is the recommended way for title pages in typst? Should I treat the title page as a regular page? For example, I would like to create the following title page. How should I approach this?

![image](https://github.com/typst/typst/assets/26227737/ee35dbb8-8dd8-4264-aee4-362239a45736)

**Comment by l0wx** [comment]
Either utilizing a template or implementing customly `à la:
```typst
#{
    set page(numbering: {})
    set align(center)
    show heading: set block(above: 8 * 1.65em, below: 1.65em)
    show heading(level: 1): set text(2em)
    show heading(level: 2): set text(weight: "regular", style: "italic")
    …
}
```

**Comment by goyalyashpal** [comment]
i believe this page will help you a lot:

== Making a Template – Typst Documentation
https://typst.app/docs/tutorial/making-a-template/
